org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -445,6 +472,48 @@ function getDesiredColourState(providerSchema, resourcePayload) {
 }
 
 /**
+ * Given a Wink provider formatted Schema, calculate the
+ * desired brightness as a percentage of the current setting
+ */
+function getDesiredBrightnessState(providerSchema, resourceSchema) {
+    var state = {};
+
+    if(providerSchema.last_reading.hasOwnProperty('brightness')) {
+
+        // current brightness, already scaled
+        var brightness = providerSchema.last_reading.brightness;
+        var desired_brightness = brightness;
+
+        if (resourceSchema.hasOwnProperty('dimmingSetPercentage')) {
+
+            desired_brightness = scaleValue(resourceSchema.dimmingSetPercentage, 100, 1.0);
+
+        } else if (resourceSchema.hasOwnProperty('dimmingIncrementPercentage')) {
+
+            var incrementBy = scaleValue(resourceSchema.dimmingIncrementPercentage, 100, 1.0);
```

Comment:
Btw, don't we need to use something akin to "scaleValue" for SmartThings as well?